src/material/list/list.md

```diff
@@ -213,6 +213,10 @@ these options, including buttons and anchors.
 Always provide an accessible label for the `<mat-selection-list>` element via `aria-label` or
 `aria-labelledby` that describes the selection being made.
 
+By default `MatSelectionList` displays radio/checkmark to identify selected items. While you can
+hide the radio indicator via `hideSingleSelectionIndicator`, this makes your component less
```

Comment:
Does this disable the indicator for multi-selection lists too?

Reply:
No, it only hides it for single-selection. There is not a way to hide it for multi-selection.

We could re-word this as:
> "While you can hide the single-selection radio indicator via `hideSingleSelectionIndicator`,